### PR TITLE
provide en_US.UTF-8 locale

### PR DESCRIPTION
**Pull request by Luap99**
see https://github.com/containers/podman/pull/19635

wait for the CI there to pass so I know this works

### CODE REVIEW

**Comment by Luap99** [comment]
@cevich PTAL

**Review by cevich — APPROVED** [review]
Changes are fine, looks like a small problem is affecting the debian SID build.  I'm not sure what [the windows error](https://cirrus-ci.com/task/6444024303714304) is (I tried [re-running it](https://cirrus-ci.com/task/4963374241415168)), hopefully it was a flake.

File: cache_images/debian_setup.sh [hunk 1]
```diff
@@ -17,6 +17,10 @@ fi
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+# Generate en_US.UTF-8 locale as this is required for a podman test (https://github.com/containers/podman/pull/19635).
+sed -i '/en_US.UTF-8/s/^#//g' /etc/locale.gen
+locale-gen
```
[the remark this answers is not in the document]
Comment (Luap99) [reply cache_images/debian_setup.sh]: thanks, done

**Comment by cevich** [comment]
@Luap99 assuming the builds work, do you want me to release the images as an update to all repos?  The alternative is you just update the value in podman's `.cirrus.yml`.

**Comment by Luap99** [comment]
I have to manually update anyway in podman because we should revert this locale fix in the podman setup script.

**Pull request merged**